src/JsonSchema/Constraints/ObjectConstraint.php

```diff
@@ -170,6 +175,26 @@ protected function &getProperty(&$element, $property, $fallback = null)
     }
 
     /**
+     * set and return a property on an object or array
+     *
+     * @param mixed $element Element to manipulate
+     * @param string $property  Property to set
+     * @param mixed $value Value to set
+     */
+    protected function &setProperty(&$element, $property, $value)
```

Comment:
Why is it necessary to return a reference? Why is it necessary to return anything at all? `$value` is passed in, so presumably whoever is calling this method already has what it needs.